app/javascript/packs/admin/overview.js

```diff
@@ -0,0 +1,144 @@
+import Chart from 'chart.js/auto';
+
+// General graph configuration
+const options = {
+  responsive: true,
+  elements: {
+    point: {
+      radius: 3,
+      hitRadius: 20,
+      hoverBorderWidth: 4,
+    },
+  },
+  layout: {
+    padding: 0,
+  },
+  plugins: {
+    legend: false,
+    tooltip: {
+      mode: 'nearest',
+      intersect: true,
+      position: 'nearest',
+    },
+  },
+  scales: {
+    x: {
+      ticks: {
+        display: false,
+      },
+      grid: {
+        display: false,
+        drawBorder: false,
+        drawOnChartArea: false,
+        drawTicks: false,
+      },
+    },
+    y: {
+      ticks: {
+        display: false,
+      },
+      beginAtZero: true,
+      grid: {
+        display: false,
+        drawBorder: false,
+        drawOnChartArea: false,
+        drawTicks: false,
+      },
+    },
+  },
+};
+
+const datasetOptions = {
+  fill: true,
+  tension: 0.5,
+  borderWidth: 2,
+};
+
+// Color sets depending on graph trend
+const positiveColors = {
+  backgroundColor: '#ECFDF5',
+  borderColor: '#10B981',
+};
+const negativeColors = {
+  backgroundColor: '#FEF2F2',
+  borderColor: '#DC2626',
+};
+
+// This is data for "Last 7 days". Obviously it has to be dynamic.
+const labels = [
+  'Mar 21',
+  'Mar 22',
+  'Mar 23',
+  'Mar 24',
+  'Mar 25',
+  'Mar 26',
+  'Mar 27',
+];
```

Comment:
labels should obviously be dependent on chosen period of time and needs to be dynamic...